### PR TITLE
[IMP] set as python library to be able to install it from pypi with pip

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

**Comment by moylop260** [comment]
@apple314159 
FYI https://pypi.python.org/pypi/panamadv

**Comment by apple314159** [comment]
Well, I'm extremely disappointed. I don't see how you can change the
license.
You don't have the right to do that. Please change it ASAP!

**Comment by oscarolar** [comment]
Sorry about that, fixed, It was not my intention to do that.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    # TODO: put package test requirements here
+]
+
+setup(
+    name='panamadv',
+    version='0.1.1',
+    description="Program to calculate the DV for the given RUC for Panama.",
+    long_description=readme + '\n\n' + history,
+    author="Jose Vasconcellos",
+    author_email='info@vauxoo.com',
```
Comment (oscarolar) [inline setup.py]: @apple314159 can you tell provide me a vali e-mail addres to put it in here?

Comment (apple314159) [inline setup.py]: First off, thank you for addressing the license issue and I'm sorry if I've
overreacted.
For the email, please use vasmac+github@gmail.com.

I'll update the copyright issue this weekend.

On Thu, Sep 3, 2015 at 8:14 PM, Olar Alca [Vauxoo] <notifications@github.com

Comment (oscarolar) [inline setup.py]: No problem, now fixed, if you dont have any other concern merge when you consider. 
thanks for your great contribution!

**Comment by moylop260** [comment]
@apple314159
What do you think of this PR?

FYI we can add a auto pip upload script by each `git push` with travis-ci but we need have permissions to write in this project (to generate [encrypt environment](http://docs.travis-ci.com/user/encryption-keys/) and enable travis-ci).

We are maintainers of other communities:
 - https://github.com/oca/
 - https://github.com/vauxoo
 - https://github.com/odoo-mexico

...and we know respect no-push directly, only by PR changes approved by you.
But this allow it install your library with:
`pip install panama-dv==GIT_TAG.GIT_NUM_COMMITS`
More info here: http://docs.openstack.org/developer/pbr/
And here: http://docs.travis-ci.com/user/deployment/pypi/

**Comment by moylop260** [comment]
@oscarolar 
The build folders should be delete to minimize the pr changes and auto create build folder from travis...

**Comment by apple314159** [comment]
When I go to https://pypi.python.org/pypi/panamadv/ I see that you listed
the project with a BSD license.
My project is distributed with an Apache license (see LICENSE file). I've
already communicated with you
on this issue and you've agreed to change it. You keep requesting changes
but you haven't done what
you said you would do.

**Comment by moylop260** [comment]
You are right!
Sorry for that.

Changed
https://github.com/apple314159/panama-dv

Please, don't merge this PR, I will work with travis, coveralls, pep8... (If you are agree with me)

**Comment by moylop260** [comment]
This pr deprecated by https://github.com/apple314159/panama-dv/pull/6

**Pull request closed without merging**